doc/manual/de/config/structure-tile/components/chart.rst

```diff
@@ -34,14 +34,94 @@ einen Zeitpunkt im Chart wird als Tooltip angezeigt, wenn man mit dem Mauszeiger
 Damit der Chart eine gewisse Größe erreicht wird hier eine Kachel mit doppelter Breite ``<cv-tile size="2x1">`` benutzt
 und der Chart selbst soll die gesamte Kachel ausfüllen ``<cv-chart ... colspan="3" rowspan="3">``.
 
+Konfiguration der Datenquelle
+#############################
+
+Im ``src``-Attribute des ``dataset``-Elements wird eine URL angegeben die die Datenquelle definiert. Bisher werden
+openHAB (`Persistence <https://www.openhab.org/docs/configuration/persistence.html>`_), InfluxDB und RRD als Datenquellen unterstützt.
+
+**Konfiguration einer openHAB-Quelle**
+
+``openhab://<Item-Name>`` wobei <Item-Name> ein beliebiger Item-Name in openHAB sein kann, für den historische Daten aufgezeichnet werden.
+Es ist zusätzlich möglich einen spezielle Persistence-Service anzugeben, falls man mehrere benutzt:
+``openhab://<service-id>@<Item-Name>``, gibt man diesen nicht an wird der in openHAB konfigurierte default-Persistence Service benutzt.
+
+**Konfiguration einer InfluxDB-Quelle**
+
+``flux://<organization>@<bucket>/<measurement>/<field>`` Optional können Angaben für die
+`aggregationWindow-Funktion <https://docs.influxdata.com/flux/v0.x/stdlib/universe/aggregatewindow/>`_ als
+Query-Parameter angehängt werden. Diese benötigen den Präfix ``ag-``. Beispiel:
+``flux://cometvisu@cv-bucket/Temperature/value?ag-every=1d&amp;ag-fn=mean`` würde in folgende Flux-Query an die
```

Comment:
Double-Check: is `&amp` correct in this case - or should it just be `&`?

Reply:
Just `&` is no valid XML (even in attribute values), it must be `&amp`